app/views/admin/proposals/index.html.erb

```diff
@@ -8,6 +8,7 @@
   <%= render Admin::SearchComponent.new(label: t("admin.shared.search.label.proposals")) %>
 
   <h3><%= page_entries_info @proposals %></h3>
+  <%= render Admin::Proposals::ProposalsComponent.new(proposals: @proposals) %>
```

Comment:
For consistency with how it’s done in the Budget Investments index, maybe we could render the download link above the `<h3>` instead?

```suggestion
  <%= render Admin::Proposals::ProposalsComponent.new(proposals: @proposals) %>
  <h3><%= page_entries_info @proposals %></h3>
```

That way, the CSV links would appear in the same position on both screens, which might feel more visually consistent.